exp-player/addon/components/exp-card-sort/component.js

```diff
@@ -168,6 +169,12 @@ export default ExpFrameBaseComponent.extend({
         }
         return responses;
     }).volatile(),
+
+    allowNext: Ember.computed('cards.[]', function() {
+        console.log(this.get('cards').length);
+      return this.get('cards').length === 0 || !config.validate;
```

Comment:
Should this depend on two conditions both being met? Using "or", it seems that there is a situation where only the first condition would need to be satisfied for this to return true.

(In other words, the `config.validate` flag might be ignored)